nvbench/device_manager.cu

```diff
@@ -16,12 +16,12 @@
  *  limitations under the License.
  */
 
-#include <nvbench/device_manager.cuh>
+#include <cuda_runtime_api.h>
```

Comment:
nit: NVBench puts the corresponding header as the first include, followed by a newline, for each translation unit. The other include changes here are fine, but let's leave the ordering as-is for consistently until we restyle the entire project.

Reply:
Returned old headers order. Please, file an issue to teach clang-format the expected headers order.